packages/client/components/AddTeamDialog.tsx

```diff
@@ -0,0 +1,210 @@
+import React, {useState} from 'react'
+import {PreloadedQuery, usePreloadedQuery, useFragment} from 'react-relay'
+import FlatPrimaryButton from './FlatPrimaryButton'
+import {Dialog} from '../ui/Dialog/Dialog'
+import {DialogContent} from '../ui/Dialog/DialogContent'
+import {DialogTitle} from '../ui/Dialog/DialogTitle'
+import {DialogActions} from '../ui/Dialog/DialogActions'
+import {Select} from '../ui/Select/Select'
+import {SelectItem} from '../ui/Select/SelectItem'
+import {SelectTrigger} from '../ui/Select/SelectTrigger'
+import {SelectGroup} from '../ui/Select/SelectGroup'
+import {SelectValue} from '../ui/Select/SelectValue'
+import {SelectContent} from '../ui/Select/SelectContent'
+import graphql from 'babel-plugin-relay/macro'
+import {AddTeamDialogQuery} from '../__generated__/AddTeamDialogQuery.graphql'
+import {AddTeamDialog_viewer$key} from '../__generated__/AddTeamDialog_viewer.graphql'
+import useMutationProps from '../hooks/useMutationProps'
+import useAtmosphere from '../hooks/useAtmosphere'
+import {AdhocTeamMultiSelect, Option} from '../components/AdhocTeamMultiSelect/AdhocTeamMultiSelect'
+import {Input} from '../ui/Input/Input'
+import AddTeamMutation from '~/mutations/AddTeamMutation'
+import useRouter from '~/hooks/useRouter'
+import getGraphQLError from '~/utils/relay/getGraphQLError'
+import SendClientSegmentEventMutation from '~/mutations/SendClientSegmentEventMutation'
+
+interface Props {
+  isOpen: boolean
+  onClose: () => void
+  onAddTeam: (teamId: string) => void
+  queryRef: PreloadedQuery<AddTeamDialogQuery>
+}
+
+const AddTeamDialogViewerFragment = graphql`
+  fragment AddTeamDialog_viewer on User {
+    ...AdhocTeamMultiSelect_viewer
+    organizations {
+      id
+      name
+    }
+  }
+`
+
+const query = graphql`
+  query AddTeamDialogQuery {
+    viewer {
+      ...AddTeamDialog_viewer
+    }
+  }
+`
+
+const labelStyles = `text-left text-sm font-semibold mb-3`
+const fieldsetStyles = `mx-0 mb-6 flex flex-col w-full p-0`
+
+const AddTeamDialog = (props: Props) => {
+  const {isOpen, onClose, queryRef, onAddTeam} = props
+  const atmosphere = useAtmosphere()
+
+  const {submitting, onCompleted, onError, error, submitMutation} = useMutationProps()
+  const {history} = useRouter()
+
+  const data = usePreloadedQuery<AddTeamDialogQuery>(query, queryRef)
+  const viewer = useFragment<AddTeamDialog_viewer$key>(AddTeamDialogViewerFragment, data.viewer)
+  const {organizations: viewerOrganizations} = viewer
+
+  const [selectedUsers, setSelectedUsers] = useState<Option[]>([])
+  const [mutualOrgsIds, setMutualOrgsIds] = useState<string[]>([])
+
+  const showOrgPicker = !!(
+    selectedUsers.length &&
+    (mutualOrgsIds.length > 1 || !mutualOrgsIds.length)
+  )
```

Comment:
-1 This shows the org picker if I invite by email, even if I only have one org to start with
<img width="696" alt="Screenshot 2023-10-05 at 12 11 59" src="https://github.com/ParabolInc/parabol/assets/7331043/021991ef-8cd6-4163-be42-b7c75f8a8996">